arc.tex

```diff
@@ -0,0 +1,110 @@
+%ENclave text from KT  https://docs.google.com/document/d/1apNSWtIpbS7aCitaF_K0JXLAAY9PvCEmmvnNvHmecdk/edit#heading=h.nwgy0freb3zv
+
+
+\section{System architecture} \label{sec:arc}
+The overall system architecture is available in \citeds{LDM-148}.
+Details on the \gls{USDF} specifications are given in \citeds{DMTN-189}.
+\action{GS,FH}{March 2022}{Should we have Specs like \gls{DMTN}-189 for UK and FRdF ?}
+
+\secref{sec:desc} gives a high level overview of the system, architecturally we look
+at this as a set of enclaves.
+As images are processed in the Prompt and Offline Production enclaves, their resulting data products are stored in the \gls{Archive} enclave and made available to the DAC enclave where data rights holders can access and analyze them.
+In addition, Rubin Observatory staff will use the Development/Integration enclave to maintain the Observatory's software tools and systems and to develop new versions of them.
+
+These enclaves are further described here and for each a series of subsections explore :
+
+
+\begin{enumerate}
+\item Threats and Security infrastructure
+\item Disaster recovery
+\end{enumerate}
+
+\subsection{Prompt \gls{Enclave}}
+
+The Prompt enclave receives images from the Observatory facilities in Chile via a Long Haul Network connection.
+It stores these and processes them into Prompt data products of three main types:
+alerts for things that have moved or changed, measurement catalogs, and processed images.
+The alerts can be further subdivided into \emph{streak} alerts for objects that have moved a long distance and \emph{non-streak} alerts for all other objects.
+Measurements in the catalogs follow the same subdivision. Images may be \gls{Commissioning} images used for testing and characterizing the Observatory systems, normal science images without significant \emph{streaks}, or delayed science images that do contain significant \emph{streaks}.
+
+\emph{Streak} alerts are transmitted to an \gls{Alert} Vetting System (AVS) located at a Trusted Broker Facility at  Lawrence Livermore National Lab.
+\emph{Non-streak} alerts and \emph{streak} alerts approved by the \gls{AVS} are to be published to the world at large within 60 seconds of the original raw image being taken.
+Normal science images are made available to data rights holders in the \gls{DAC} after an 80 hour embargo period.
+Delayed science images, as identified by the \gls{AVS}, and any unapproved \emph{streak} measurements taken from them are made available after a 10 day embargo period. Commissioning images are made available to data rights holders after a 30 day embargo period.
+
+All Prompt data products are checked for quality by automated systems but also by human operators from the Rubin Observatory staff, who have access to all images and data products in order to perform this function.
```

Comment:
This could create the impression that humans look at everything. Maybe insert "can be spot-checked"?

Reply:
Sure.  Most likely they'll monitor summary statistics and spot-check random samples.  The main point here was that staff need to have access to anything at any time, not pre-selected samples or something like that.